app/data_providers/snowplow/config.py

```diff
@@ -21,7 +21,7 @@ class SnowplowConfig:
     BUFFER_SIZE: int = 1  # Number of events to buffer before flushing to Snowplow. 1 = No buffer.
 
     CORPUS_SLATE_SCHEMA = 'iglu:com.pocket/corpus_slate/jsonschema/3-0-0'
-    CORPUS_SLATE_LINEUP_SCHEMA = 'iglu:com.pocket/corpus_slate_lineup/jsonschema/1-0-11'
+    CORPUS_SLATE_LINEUP_SCHEMA = 'iglu:com.pocket/corpus_slate_lineup/jsonschema/2-0-0'
```

Comment:
This seems right and aligns with the new schema version in the schema change: https://github.com/Pocket/spec/pull/181